apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -166,4 +169,10 @@ Page {
         key: "/apps/sailfish-browser/settings/home_page"
         defaultValue: "http://jolla.com/"
     }
+
+    Connections {
+        target: SearchEngineModel
+
+        onInstalled: searchEngineConfig.value = title
```

Comment:
Even this could be written to the dconf value directly on the C++ side. However, I think it's good already like this.

This Connections can be improved in another task. Will file one.